spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/oauth/DataflowOAuthIT.java

```diff
@@ -92,4 +92,60 @@ void securedSetup() throws Exception {
 			}
 		}
 	}
+
+	@Test
+	void securedSetupKeycloak() throws Exception {
+		log.info("Running securedSetupKeycloak()");
+		this.dataflowCluster.startIdentityProvider(TagNames.KEYCLOAK_26);
```

Comment:
What is the significance of KEYCLOAK_26?

Reply:
KEYCLOAK 26 is latest release. I tested with bothh 25.0 and 26.0 to make sure we cover the last 2 versions.